dendrite/pipeline.yml

```diff
@@ -35,6 +35,22 @@ steps:
       - docker#v3.0.1:
           image: "golang:1.13"
           mount-buildkite-agent: false
+
+  - command:
+      - "curl -fsSL https://get.docker.com -o get-docker.sh"
+      - "sh get-docker.sh"
+      - "bash build/scripts/complement.sh"
```

Comment:
I agree that we don't want to pull this script from the repository - for the small number of lines, let's inline this.